android/buildAndroidBOINC-CI.sh

```diff
@@ -268,6 +259,7 @@ if [ $build_with_vcpkg = "yes" ]; then
     else
         triplets_setup="default"
     fi
+    packeges="rappture curl[core,openssl]"
```

Comment:
Let's fix this typo:
`packeges` -> `packages`

Reply:
Thank you! I fix the typo.